cpp/src/gandiva/function_registry_common.h

```diff
@@ -190,7 +190,7 @@ typedef std::unordered_map<const FunctionSignature*, const NativeFunction*, KeyH
 #define NUMERIC_TYPES(INNER, NAME)                                                       \
   INNER(NAME, int8), INNER(NAME, int16), INNER(NAME, int32), INNER(NAME, int64),         \
       INNER(NAME, uint8), INNER(NAME, uint16), INNER(NAME, uint32), INNER(NAME, uint64), \
-      INNER(NAME, float32), INNER(NAME, float64)
+      INNER(NAME, float32), INNER(NAME, float64), INNER(NAME, decimal128)
```

Comment:
this is a bit risky - can you please check if all of fns with this macro are defined for decimals ?

Reply:
I implemented additional decimal functions(isnull, isnotnull, is_distinct, is_not_distinct, isnumeric) which will be added to the registry when we include decimal128 in numeric types.